app/editor/src/features/navbar/NavBar.tsx

```diff
@@ -169,10 +169,22 @@ export const NavBar: React.FC = () => {
 
           {/* Content Configuration
               items:
-              Tags，Media Types，Columnists & Pundits，Ministers，Shows & Programs
+              Tags，Media Types，Columnists & Pundits，Ministers，Shows & Programs, Actions
```

Comment:
Moved several menu items around to group them with related functionality.  Also added back the work orders admin section.